### DISCUSSION TITLE
Странно работает кнопка BACKSPACE

### DISCUSSION THREAD

**Discussion by CROGOT** [codewars/codewars.com#3068] [Q&A]
В браузере Chrome редактирую код с планшета на андройде экранной клавиатурой и когда пробую стирать символы и дохожу до начала строки, то по логике курсор должен перейти на строку выше, но у меня просто клавиатура прячется, типа символы кончились и редактировать нечего.
Хотя в других места удаление работает правильно (даже в данном месте - проверил).
Как сделать чтобы можно было нормально удалять?
P. S. С выделением текста тоже непонятки, не смог нормально выделить (я думаю эти проблемы связаны)

**Comment by W0uldYouKindly** [comment]
Would be nice if you could write this in English.
